src/components/markdown-renderer/markdown-renderer.scss

```diff
@@ -1,4 +1,4 @@
-/*
+/*!
```

Comment:
I think this is an error?

Reply:
I guess not?.. The webstorm auto-reformat inserted this character. :shrug:

Reply:
Well it's in a comment, so quite safe to ignore.